packages/schematics/angular/universal/files/src/__main@stripTsExtension__.ts.template

```diff
@@ -7,3 +7,4 @@ if (environment.production) {
 }
 
 export { <%= rootModuleClassName %> } from './app/<%= stripTsExtension(rootModuleFileName) %>';
+export { renderModule } from '@angular/platform-server';
```

Comment:
I am considering adding `renderModuleFactory` here to make it easier for prerender and app-shell ivy opt-out

//cc @MarkPieszak